pennylane/plugins/tests/conftest.py

```diff
@@ -209,7 +251,7 @@ def pytest_runtest_makereport(item, call):
             # and those using not implemented features
             if (
                 call.excinfo.type == qml.DeviceError
-                and "not supported on device" in str(call.excinfo.value)
+                and "supported" in str(call.excinfo.value)

```

Comment:
How come this needed a change?

Reply:
I noticed that not all of the plugins were standardized in their error messages for unsupported behaviour :( For example, the Cirq plugin raises an 'unsupported' error if you attempt to use certain gates it some situations.

Reply:
Oh, I see! Hmm, would it be worth adding an extra condition here with `or`?